bin/diagnostic.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+const ask = require('../lib/ask.js');
+
+let answer = 'not empty';
+
+while (answer !== '') {
+  answer = ask("Guess my secret? ");
+  if (answer === '7') {
```

Comment:
What happens if my guess is 6? It doesn't meet any of the criteria. Almost though!